### ISSUE TITLE
Prediction of sale prices all zeroes

### ISSUE THREAD

**Issue by dreamk73** [yashu-seth/pytorch-tabular#1]
I am trying to learn more about pytorch and in particular about using categorical and continuous tabular data. I am trying to run your code, but for some reason when I am trying to predict new sale prices for the test set, it is generating all zeros. The training loss also seems to be very high in my case. Could you maybe share some results from your code and some code to return the predicted sale prices for the test set? I would like to figure out what is going wrong in my case.
